Dalamud.Injector/GameStart.cs

```diff
@@ -64,6 +69,63 @@ public static Process LaunchGame(string workingDir, string exePath, string argum
                 Marshal.StructureToPtr(secDesc, psecDesc, true);
             }
 
+            var procAttrListBuffer = nint.Zero;
+            var parentProcessHandle = nint.Zero;
+            if (!dontUnelevate)
+            {
+                try
+                {
+                    if (PInvoke.GetWindowThreadProcessId(PInvoke.GetShellWindow(), out var shellProcessId) == 0)
+                        Marshal.ThrowExceptionForHR(Marshal.GetHRForLastWin32Error());
```

Comment:
See comment in PR chain - can this (gracefully) stop the unelevation process and/or somehow not get run at all on *nix operating systems?